### PR TITLE
CA-767 Hiding grid view on mobile

### PR DESCRIPTION
**Pull request by Galzk**

### CODE REVIEW

File: src/components/MenuBar/MenuBar.tsx [hunk 1]
```diff
@@ -24,7 +23,6 @@ import { ParticipantInformation } from 'state';
 import useIsHostIn from '../../hooks/useIsHostIn/useIsHostIn';
 import usePublishDataTrack from '../../hooks/useDataTrackPublisher/useDataTrackPublisher';
 import useDataTrackListener from '../../hooks/useDataTrackListener/useDataTrackListener';
-// import { LogglyTracker } from 'react-native-loggly-jslogger';
```
Comment (talr) [inline src/components/MenuBar/MenuBar.tsx:27]: I suggest to uncomment it and make very basic usages of Loggly until we adopt eScribers TypeScript Logger to a full extent.

File: src/components/MenuBar/MenuBar.tsx [hunk 2]
```diff
@@ -239,9 +237,6 @@ export default function MenuBar() {
         )}
         <div className={classes.rightButtonContainer}>
           <ToggleGridViewButton />
-          {/* {!mobileAndTabletCheck() && (
-            <SettingsButton selectedAudioDevice={selectedAudioDevice} selectedVideoDevice={selectedVideoDevice} />
-          )} */}
```
Comment (talr) [inline src/components/MenuBar/MenuBar.tsx]: Please check how the Setting to Mobile is done in the original Repository / Upstream.

File: src/components/MenuBar/MenuBar.tsx [hunk 3]
```diff
@@ -250,35 +245,3 @@ export default function MenuBar() {
     </AppBar>
   );
 }
-// function detectBrowser() {
```
Comment (talr) [inline src/components/MenuBar/MenuBar.tsx:253]: Please make sure it is sent to Loggly

File: src/hooks/useWindowSize/useWindowSize.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
```
Comment (talr) [inline src/hooks/useWindowSize/useWindowSize.ts:1]: Please include link to the original library

File: src/components/MenuBar/MenuBar.tsx (hunk 3 above)
Comment (talr) [reply src/components/MenuBar/MenuBar.tsx:253]: ?

File: src/components/MenuBar/MenuBar.tsx (hunk 2 above)
Comment (talr) [reply src/components/MenuBar/MenuBar.tsx]: ?

File: src/hooks/useWindowSize/useWindowSize.ts (hunk 4 above)
Comment (talr) [reply src/hooks/useWindowSize/useWindowSize.ts:1]: ?

**Review by talr — CHANGES_REQUESTED** [review]

File: src/components/MenuBar/MenuBar.tsx (hunk 2 above)
Comment (Galzk) [reply src/components/MenuBar/MenuBar.tsx]: It is not done on upstream
We should decide on how we're going to approach it, I'll open a ticket about that

Comment (Galzk) [reply src/components/MenuBar/MenuBar.tsx]: Please take a look:
https://escribers.atlassian.net/browse/CA-771

**Review by talr — APPROVED** [review]

**Pull request merged**